pkg/e2e/operators/machine-api-operator.go

```diff
@@ -8,158 +8,69 @@ import (
 	"github.com/golang/glog"
 	g "github.com/onsi/ginkgo"
 	o "github.com/onsi/gomega"
-	osconfigv1 "github.com/openshift/api/config/v1"
 	e2e "github.com/openshift/cluster-api-actuator-pkg/pkg/e2e/framework"
-	cvoresourcemerge "github.com/openshift/cluster-version-operator/lib/resourcemerge"
 	kappsapi "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/wait"
-	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var _ = g.Describe("[Feature:Operators] Machine API operator should", func() {
+var _ = g.Describe("[Feature:Operators] Machine API operator deployment should", func() {
 	defer g.GinkgoRecover()
 
 	g.It("be available", func() {
 		var err error
 		client, err := e2e.LoadClient()
 		o.Expect(err).NotTo(o.HaveOccurred())
-
-		key := types.NamespacedName{
-			Namespace: e2e.TestContext.MachineApiNamespace,
-			Name:      "machine-api-operator",
-		}
-		d := &kappsapi.Deployment{}
-
-		err = wait.PollImmediate(1*time.Second, e2e.WaitShort, func() (bool, error) {
-			if err := client.Get(context.TODO(), key, d); err != nil {
-				glog.Errorf("error querying api for Deployment object: %v, retrying...", err)
-				return false, nil
-			}
-			glog.Infof("Have %v ready replicas, expecting 1", d.Status.ReadyReplicas)
-			if d.Status.ReadyReplicas < 1 {
-				return false, nil
-			}
-			return true, nil
-		})
-		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(isDeploymentAvailable(client, "machine-api-operator")).To(o.BeTrue())
 	})
 
 	g.It("reconcile controllers deployment", func() {
 		var err error
 		client, err := e2e.LoadClient()
 		o.Expect(err).NotTo(o.HaveOccurred())
 
+		controllersDeployment := "clusterapi-manager-controllers"
 		key := types.NamespacedName{
```

Comment:
I think this would be better if we substituted `key` for `controllersDeployment`. Then where we consume/log we can use `controllersDeployment.Name`, et al.